python/python/tests/test_async_client.py

```diff
@@ -2291,6 +2291,308 @@ async def test_geosearch_no_result(self, redis_client: TRedisClient):
                 GeoSearchByBox(10, 10, GeoUnit.MILES),
             )
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_geosearchstore_by_box(self, redis_client: TRedisClient):
```

Comment:
is it worthwhile adding tests to compare beyond the dateline?  Or up near the poles.  

Right now, our tests are focused around Europe, which isn't anywhere close to the Pacific coast datelines, and it close to the equator. I'm not sure how much we want to stress the system by doing geosearch around something like Tonga Island (-21.178986, -175.198242).  If you do a large enough search, you should get islands on the other side of the meridian, like Fiji (-17.713371, 178.065033).  

Of course, this kind of test is more testing Redis, rather than our API.  So this is optional.

Reply:
maybe we can add it as a post GA small task